frostsnapp/native/src/coordinator.rs

```diff
@@ -559,36 +566,204 @@ impl FfiCoordinator {
+                            .outputs()
+                            .iter()
+                            .enumerate()
+                            .map(|(vout, output)| api::TxOutInfo {
+                                vout: vout as _,
+                                amount: output.value,
+                                script_pubkey: RustOpaque::new(output.owner().spk()),
+                                is_mine: match output.owner().local_owner_key() {
+                                    Some(this_appkey) => this_appkey == master_appkey,
+                                    None => false,
+                                },
+                            })
+                            .collect(),
+                    },
+                }
+            }
+        };
+        Some(api::DetailedSigningState { state, details })
+    }
+
+    pub fn active_signing_session(
+        &self,
+        session_id: SignSessionId,
+    ) -> SyncReturn<Option<api::DetailedSigningState>> {
+        let coord = self.coordinator.lock().unwrap();
+        SyncReturn(|| -> Option<api::DetailedSigningState> {
+            let session = coord.active_signing_sessions().get(&session_id)?;
+            let master_appkey = coord
+                .get_frost_key(session.key_id)?
+                .complete_key
+                .as_ref()?
+                .master_appkey;
+            Self::_active_signing_session_details(
+                master_appkey,
+                session_id,
+                &session.init,
+                session.received_from(),
+            )
+        }())
+    }
+
+    pub fn active_signing_sessions(
+        &self,
+        key_id: KeyId,
+    ) -> SyncReturn<Vec<api::DetailedSigningState>> {
+        let coord = self.coordinator.lock().unwrap();
+        let sessions = coord
+            .active_signing_sessions()
+            .iter()
+            .inspect(|&(&id, _)| event!(Level::DEBUG, "Found active signing session: {}", id))
+            .filter(|(_, session)| session.key_id == key_id)
+            .filter_map(|(id, session)| -> Option<api::DetailedSigningState> {
+                Self::_active_signing_session_details(
+                    coord
+                        .get_frost_key(key_id)?
+                        .complete_key
+                        .as_ref()?
+                        .master_appkey,
+                    *id,
+                    &session.init,
+                    session.received_from(),
+                )
+            });
+        SyncReturn(sessions.collect())
+    }
+
+    pub fn unbroadcasted_txs(
+        &self,
+        super_wallet: api::SuperWallet,
+        key_id: KeyId,
+    ) -> SyncReturn<Vec<api::SignedTxDetails>> {
+        let coord = self.coordinator.lock().unwrap();
+        let super_wallet = &*super_wallet.inner.lock().unwrap();
+        let txs = coord
+            .finished_signing_sessions()
+            .iter()
+            .filter(|(_, session)| dbg!(session.key_id == key_id))
```

Comment:
Noticed two debugs here, maybe can remove after development, or use `event!` like you have elsewhere for the wildcard one at the end of the match below